charts/metallb/templates/podmonitor.yaml

```diff
@@ -75,6 +75,7 @@ spec:
 {{- toYaml .Values.prometheus.podMonitor.relabelings | nindent 4 }}
 {{- end }}
 ---
+{{- if and .Values.prometheus.rbacPrometheus .Values.prometheus.podMonitor.enabled }}
```

Comment:
the whole file is under `.Values.prometheus.podMonitor.enabled`, so I think this one is redundant

Reply:
👍 thanks, I didn't notice that..